src/stable/lib/globals.ts

```diff
@@ -12,42 +12,51 @@ import { Action } from './state';
 
 declare global {
     // eslint-disable-next-line no-var
-    var _azleActions: Action[];
+    var _azleActions: Action[] | undefined;
     // eslint-disable-next-line no-var
     var _azleCanisterClassMeta: CanisterClassMeta | undefined;
     // eslint-disable-next-line no-var
-    var _azleCanisterMethodNames: { [key: string]: string };
+    var _azleCanisterMethodNames: { [key: string]: string } | undefined;
     // eslint-disable-next-line no-var
-    var _azleDispatch: (action: Action) => void;
+    var _azleDispatch: ((action: Action) => void) | undefined;
     // eslint-disable-next-line no-var
-    var _azleExperimental: boolean;
+    var _azleExperimental: boolean | undefined;
     // eslint-disable-next-line no-var
     var _azleIcExperimental: AzleIcExperimental | undefined;
     // eslint-disable-next-line no-var
-    var _azleIcpReplicaWasmEnvironment: boolean;
+    var _azleIcpReplicaWasmEnvironment: boolean | undefined;
     // eslint-disable-next-line no-var
     var _azleIc: AzleIc | undefined;
     // eslint-disable-next-line no-var
-    var _azleInitCalled: boolean;
+    var _azleInitCalled: boolean | undefined;
```

Comment:
examples/experimental/test/end_to_end/http_server/multi_deploy/src/index.ts are we okay with this maybe returning undefined? It's an http response so it's not exactly strongly typed to begin with... and it's experimental

Reply:
Same with examples/experimental/test/end_to_end/http_server/multi_deploy/src/index.ts

Reply:
Same with examples/experimental/test/end_to_end/http_server/sqlite_drizzle/src/server/index.ts

Reply:
Same with examples/experimental/test/end_to_end/http_server/sqlite_typeorm/src/server/index.ts

Reply:
DIFFERENT issue, same line
examples/stable/test/end_to_end/candid_rpc/multi_deploy/src/index.ts needs to be updated

Reply:
I think the http_server ones are good for now, but I have fixed the candid_rpc multi_deploy one